flexget/plugins/input/sonarr_emit.py

```diff
@@ -115,15 +116,8 @@ def on_task_input(self, task, config):
                     # Test mode logging
                     if entry and task.options.test:
                         log.info("Test mode. Entry includes:")
-                        log.info("    Title: %s" % entry["title"])
-                        log.info("    URL: %s" % entry["url"])
-                        log.info("    Show name: %s" % entry["series_name"])
-                        log.info("    TVDB ID: %s" % entry["tvdb_id"])
-                        log.info("    TVRAGE ID: %s" % entry["tvrage_id"])
-                        log.info("    Season: %s" % entry["series_season"])
-                        log.info("    Episode: %s" % entry["series_episode"])
-                        log.info("    Series ID: %s" % entry["series_id"])
-                    continue
+                        for key, value in entry.items():
+                            log.info('     %s: %s' % (key.capitalize(), value))
```

Comment:
I don't think this should be info level. verbose, or even debug is probably better don't you think?